src/libsync/discovery.cpp

```diff
@@ -498,6 +498,7 @@ void ProcessDirectoryJob::processFile(PathTuple path,
 
     if (dbEntry._modtime == localEntry.modtime && dbEntry._type == ItemTypeVirtualFile && localEntry.type == ItemTypeFile) {
         item->_type = ItemTypeFile;
+        qCWarning(lcDisco) << "Changing item type from virtual to normal file";
```

Comment:
general question
why warning level and not just info level ?
I would tend to think that those code decisions however strange they seem to be are probably done on purpose (that was my understanding)